### ISSUE TITLE
Hyperparameter optimization does not return learning rate

### ISSUE THREAD

**Issue by AlexMRuch** [jdb78/pytorch-forecasting#104]
Presently, the hyperparameter optimization returns the best parameters for
```
{'gradient_clip_val': 1.028043566346387,
 'hidden_size': 176,
 'dropout': 0.22095396475678628,
 'hidden_continuous_size': 36,
 'attention_head_size': 1}
```
even when `use_learning_rate_finder=True`. I expect this could be fixed easily just by adding it to the optuna parameters to track.

**Comment by AlexMRuch** [comment]
Also, is there a way to limit the printouts for the hyperparameter optimization trials? If one is running 100 trials, they're going to have to run it as a script and not in a notebook lest they want to blow up their notebook with all the intermediate output. Would be nice to have a two-way tqdm-style tracker with the top bar going from 0-100 trials and with the bottom bar looping through the number of epochs per trial.

**Comment by AlexMRuch** [comment]
Lastly, thanks a ton for adding in that hyperparameter optimization! I'm a big fan of `optuna`, and think this is a really great addition. You might want to call out it's utility more in the docs/tutorial for folks who may be new to DL for time series and who aren't sure how to best set things about hidden levels and attention heads a priori – let the model itself find the best settings for the task at hand! 😄

**Comment by jdb78** [comment]
Great points! I am addressing them now in #106

**Comment by AlexMRuch** [comment]
Thanks a ton on adding those in! I'll test them out in a bit

**Comment by AlexMRuch** [comment]
Super solid job adding val_loss to the training status by the way 🎉  I had really wanted that to be shown while the model is training!

**Comment by AlexMRuch** [comment]
Also, I can confirm that the results are _good_ 😄 
```
{'gradient_clip_val': 0.1709229605211334,
 'hidden_size': 238,
 'dropout': 0.13812025470427342,
 'hidden_continuous_size': 8,
 'attention_head_size': 1,
 'learning_rate': 0.006309573444801936}
```
^^^ results from doing 3 trials with 5 epochs each (just to see if the outputs worked) 🎉

**Comment by AlexMRuch** [comment]
FYI, setting `verbose=0` still gives lots of printouts
![image](https://user-images.githubusercontent.com/22121273/96636064-c152ac00-12ea-11eb-9abd-fd987f271d1c.png)

I'm going to switch to scripts soon, but just wanted to let you know

**Comment by jdb78** [comment]
Annoying mishap. Pushed a fix.

**Comment by AlexMRuch** [comment]
It doesn't look like that push is on https://anaconda.org/conda-forge/pytorch-forecasting yet (still version 0.5.2).

**Comment by jdb78** [comment]
Yes. It's unreleased.
